### PR TITLE
feat: move NumberInput to svelte-ui

### LINKED ISSUE #9606: Make NumberInput a component of the UI library
### Is your enhancement related to a problem? Please describe

today people can use Input but they do some workaround if they have numbers

https://github.com/containers/podman-desktop-extension-ai-lab/pull/1994

we do have a component in Podman Desktop but it's not exposed

### Describe the solution you'd like

make the component part of Podman Desktop ui library

### Describe alternatives you've considered

_No response_

### Additional context

wondering if the increment/decrement should always be visible, or if it's based on a hide/show property

### PR DESCRIPTION
**Pull request by feloy**
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Move NumberInput to svelte-ui

### What issues does this PR fix or reference?

Fixes #9606 

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/ui/src/lib/inputs/NumberInput.svelte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-import { Input } from '@podman-desktop/ui-svelte';
+import { Input } from '..';
```
Comment (axel7083) [inline packages/ui/src/lib/inputs/NumberInput.svelte]: weird import ?

**Review by axel7083 — APPROVED** [review]
suggestion (follow-up PR): add a Storybook template for the NumberInput

**Review by gastoner — APPROVED** [review]

**Review by benoitf — APPROVED** [review]
needs to fix the `import { Input } from '..';` import before merging

**Review by benoitf — APPROVED** [review]
reapproving (and thanks for the fix)

**Pull request merged**